### ISSUE TITLE
Loophole in 2.5.8 Target Size (Minimum)?

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#3045]
The spacing exception has been designed with tightly packed controls in mind, but as soon as controls are far enough apart, (or, in extreme cases, there *are* no other controls around), it effectively loses any normative power to define a minimum size.

Taking the extreme opposite case: imagine a page that only has a single button, and no other actionable/clickable controls. Per the normative wording, using the spacing exception, it would appear that it's "legal" for that single button to be sized at 1 CSS px by 1 CSS px.

Just wondering if we want to acknowledge this loophole, or hope that nobody twigs? EDIT: acknowledge as in "at least mention it in the understanding, but also emphasise that while it may normatively pass, as a *best practice* authors really shouldn't do it"

**Comment by alastc** [comment]
> Per the normative wording, using the spacing exception, it would appear that it's "legal" for that single button to be sized at 1 CSS px by 1 CSS px.

Agree. The alternative would be to specify some smaller minimum value, which which case that becomes the new minimum size people think about.

I'm fairly torn on adding something about it. If we do that becomes the talking point, if we don't it becomes something that some people know about and others haven't thought about.

If we did, we would need to be heavily caveated, as it's bad for everyone.

**Comment by patrickhlauke** [comment]
as previously mentioned in a similar related conversation, i'd rather the understanding doc was very upfront about the limitations of the SC, rather than trying to hide things and hope that devs won't notice. if the explanation for the SC explicitly turned it around to "the aim is to avoid having adjacent controls that are too difficult to hit, rather than setting an absolute minimum for all targets", it then at least excuses/explains the gap here. and then we strongly suggest that as best practice, authors should also make sure their non-grouped, non-adjacent targets still have a sufficiently large size to be comfortable and easy to use.

does that weaked the SC? possibly, in the eyes of those that misunderstood it/didn't realise the consequences. but at least it's realistic (and we won't end up with some clever clogs later on after publication coming at this from a "how silly are AGWG? have they not realised this is the case? how pointless is their work?"

**Comment by davidc-gds** [comment]
Hey there! Thought I'd try and find a relevant open issue to add in a thought I've been processing for a few weeks, but I'm not sure if this is the appropriate place to put it.

**Target size (minimum) is actually about target spacing.**

The listed exception of 'spacing' for 'Target Size (Minimum)' kind of makes it so that the success criterion isn't really reliant on target size at all. It's really more about having 24px of spacing between targets.

I've anecdotally found man people confused as to how the exceptions interplay with the size aspect of this criterion, and it makes me wonder if it would be clearer to frame it form the perspective of spacing instead.

Something like '2.5.8 Target Spacing', with the main description being about having that 24px by 24px spacing between targets, with an exception for targets greater than 24px by 24px.

Might help a lot with misinterpretation, as the currently drafted '[Target Size (Minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum)' is wildly different from its '[Target Size (Enhanced)](https://www.w3.org/TR/WCAG22/#target-size-enhanced)' counterpart, which focuses a lot more explicitly on the actual size of a target, without a spacing exception that breaks the 44px size requirement.

Once again if this isn't the right place for the discussion, I can move this to a separate issue 👍

**Comment by patrickhlauke** [comment]
added a commit (https://github.com/w3c/wcag/commit/78d1253162cd22f2300d80ab627843c2048d72f4) on https://github.com/w3c/wcag/pull/3103 to address this

**Comment by GreggVan** [comment]
hmmm

Target spacing is also not quite right since that sounds like the space between targets…   and this actually allows there to be no space between targets.

And I haven’t seen any SC named after one of its exception.   

I think target size is actually a better name for it - though I see your point - and it isnt great.    

Maybe it should be Target Offset 
That would be more accurate — but I still think that  Target Size  gives people the right impression and is the driving force behind all the intricacies of this SC

gregg

------------------------------
Gregg Vanderheiden
***@***.***
